File: sdks/java/src/main/java/com/blueprint/sdk/Blueprint.java
package com.blueprint.sdk;

/**
 * Blueprint SDK for Java
 */
public class Blueprint {
    private String version;
    
    public Blueprint() {
        this.version = "1.0.0";
    }
    
    /**
     * Returns a greeting message
     */
    public String hello() {
        return "Hello from Blueprint SDK!";
    }
    
    /**
     * Returns the SDK version
     */
    public String getVersion() {
        return version;
    }
}

File: sdks/kotlin/src/main/kotlin/com/blueprint/sdk/Blueprint.kt
package com.blueprint.sdk

/**
 * Blueprint SDK for Kotlin
 */
class Blueprint {
    private val version: String = "1.0.0"
    
    /**
     * Returns a greeting message
     */
    fun hello(): String {
        return "Hello from Blueprint SDK!"
    }
    
    /**
     * Returns the SDK version
     */
    fun getVersion(): String {
        return version
    }
}
